env_setup/env_setup.yml

```diff
@@ -73,6 +73,10 @@
 
         - name: "Revert and cleanup existing snapshots on existing VM if required"
           include_tasks: ../common/vm_cleanup_snapshot.yml
+          when: cleanup_old_snapshots | bool
+
+        - name: "Check base snapshot existence and/or revert to it"
+          include_tasks: ../common/base_snapshot_check_revert.yml
```

Comment:
This task shall be executed before `vm_cleanup_snapshot.yml`. Otherwise when all snapshots are cleaned up, how to revert to base snapshot?

Reply:
As discussed offline, will move the check_vm_settings.yml to test_setup.yml